src/Server/Remote/NativeSsh.php

```diff
@@ -85,12 +85,17 @@ public function run($command)
 
         $sshCommand = 'ssh ' . implode(' ', $sshOptions) . ' ' . escapeshellarg($username . $hostname) . ' ' . escapeshellarg($command);
 
-        $process = new Process($sshCommand);
-        $process
-            ->setPty($serverConfig->getPty())
-            ->setTimeout(null)
-            ->setIdleTimeout(null)
-            ->mustRun();
+        try {
+            $process = new Process($sshCommand);
+            $process
+                ->setPty($serverConfig->getPty())
+                ->setTimeout(null)
+                ->setIdleTimeout(null)
+                ->mustRun();
+        } catch (ProcessFailedException $exception) {
+            $errorMessage = \Deployer\isVerbose() ? $exception->getMessage() : $process->getErrorOutput();
```

Comment:
@elfet Error output returns only error string like "invalid credentials" and an exception message has current look with full broken command, current directory, error string, etc. I think that common user needs only concise error string.